### PR TITLE
add ocis-reva

### PR DESCRIPTION
**Pull request by butonic**
initial integration of ocis-reva

### CODE REVIEW

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pkg/command/reva.go  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/owncloud/ocis/pullRequest?prid=4699681&bid=15704955)

File: pkg/command/reva.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	return cli.Command{
+		Name:     "reva",
+		Usage:    "Start reva server",
+		Category: "Extensions",
+		Flags:    flagset.ServerWithConfig(cfg.Reva),
+		Action: func(c *cli.Context) error {
+			scfg := configureReva(cfg)
+
+			return cli.HandleAction(
+				command.Server(scfg).Action,
```
Comment (refs) [inline pkg/command/reva.go]: just call server with `cfg.Reva`

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pkg/command/reva.go  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/owncloud/ocis/pullRequest?prid=4699681&bid=15704955)

File: pkg/command/reva.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+				c,
+			)
+		},
+	}
+}
+
+func configureReva(cfg *config.Config) *svcconfig.Config {
```
Comment (refs) [inline pkg/command/reva.go]: this is basically dead code 💀

**Comment by tboerger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pkg/command/reva.go  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/owncloud/ocis/pullRequest?prid=4699681&bid=15704955)

**Pull request merged**